### ISSUE TITLE
Request for license clarification

### ISSUE THREAD

**Issue by fraser125** [PeterLemon/N64#5]
tl;dr Do you have a preferred license for crediting, reusing, modifying or distributing the code samples in this repository.

First of all thank you for some amazing code samples, I've been reading through them for almost a year now and continue to find subtle nuances as I learn new things about the N64 as well. 

I would like to create an N64 assembly language tutorial based on some of the code etc in your repository. I don't plan on making a profit, and if you have a license preference I will respect it. I would also like to "distribute" or otherwise link to your LIB code constants and macros and N64_Bootcode.bin, hopefully in a zip file with the Bass assembler and Chksum64 utility compiled for Windows. 

For an example of what I've done so far please see [Tutorial Day 2](https://sites.google.com/site/consoleprotocols/home/n64-assembly-tutorial-day02) I started with a copy of your Initialize ROM explained each line. In this example you can see that I've clearly credited you and included a link to the relevant repository. Would you prefer I use your handle, first name or full name?

For other lessons I plan to start with some of your examples and change or tweak from there. For example [Day 4](https://sites.google.com/site/consoleprotocols/home/n64-assembly-tutorial-day04) is planned to be based on your Video Interface Setup and enhanced with a very simple colored square in the frame buffer. I'm also thinking of using some different assembly instructions to make the VI values more obvious and easier to modify. In this case I'm planning to use the term "Inspired by Peter" with a link to the original repository since the code will be pretty heavily modified.
Thanks

**Comment by PeterLemon** [comment]
Hi fraser125,

Thanks for checking out my N64 programming examples =D
I love how you are making a tutorial out of them, I was inspired by the old N64 tutorials by GbaGuy myself!

I do not have a preferred license for crediting, reusing, modifying or distributing the code samples in my repository.
I think it fine for you to link to my repository.
I like your example on Tutorial Day 2, & how you credit me e.g saying either "Pete" or "Peter Lemon" with a link to my repository.
I also like all the ideas you have put forward about the heavily modified code, good luck with all that.

Thanks for the feedback on my stuff, I made these sources to help people get started on the N64, so I am glad you are enjoying them.

**Comment by fraser125** [comment]
Thank you! They are now in the sidebar of my site so easier to find.
